backend/server/issue.go

```diff
@@ -406,14 +408,24 @@ func (s *Server) createIssue(ctx context.Context, issueCreate *api.IssueCreate,
 	if assignee == nil {
 		return nil, echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("assignee %d not found", issueCreate.AssigneeID))
 	}
+	// TODO(p0ny): remove issueCreate.Payload
```

Comment:
Issue payload is never used before.
I'm considering forbidding the frontend from modifying the issue payload directly.